library/core/src/str/mod.rs

```diff
@@ -1146,7 +1146,7 @@ impl str {
     ///
     /// assert_eq!(s.find('L'), Some(0));
     /// assert_eq!(s.find('é'), Some(14));
-    /// assert_eq!(s.find("pard"), Some(17));
+    /// assert_eq!(s.find("part"), Some(17));
```

Comment:
what leads you to believe that this should've been named "part"? this is not right.